src/components/CurrencyInputPanel/CurrencyInputPanel.component.tsx

```diff
@@ -51,6 +51,7 @@ export const CurrencyInputPanelComponent = ({
   maxAmount,
   currencyWrapperSource = CurrencyWrapperSource.SWAP,
   disableCurrencySelect = false,
+  isOutputPanel,
```

Comment:
This component triggers the prop drilling meter :)

Reply:
I agree!! This component has a lot of props but I wanted to use an existing component. Should I create a new one like `CurrencyOutputPanel` ? If you have any idea how I can improve it let me know :)